packages/components/data-table-manager/src/data-table-settings/data-table-settings.js

```diff
@@ -89,8 +95,13 @@ const DataTableSettings = (props) => {
         <div>{props.topBar}</div>
         {dropdownOptions.length > 0 && (
           <SelectContainer>
+            <AccessibleHiden>
+              <label htmlFor="table-settings-dropdown">
+                {'Open table manager dropdown'}
```

Comment:
```suggestion
                Open table manager dropdown
```

Should work too I think.